ph5/utilities/ph5validate.py

```diff
@@ -494,7 +494,12 @@ def check_station_completeness(self, station):
                 time = int(true_pickup - pickup_time)
                 warning.append("Data exists after pickup time: "
                                + str(time) + " seconds ")
+            self.ph5.forget_das_t(das_serial)
         except BaseException:
```

Comment:
Is there a reason `Exception` isn't used, or better yet the possible specific failures?

Reply:
Yes this should be `Exception`. I'll change it.